speaker/bgp_controller.go

```diff
@@ -83,48 +84,50 @@ newPeers:
 }
 
 // nodeHasHealthyEndpoint return true if this node has at least one healthy endpoint.
```

Comment:
Thanks for indulging me. It seems to have reduced the amount of code a little bit. This comment is wrong now.